llvm-spirv/test/transcoding/intel_fpga_lsu_optimized.ll

```diff
@@ -2,7 +2,7 @@
 ; SYCL source code can be found below:
 
 ; #include <CL/sycl.hpp>
-; #include <CL/sycl/intel/fpga_extensions.hpp>
+; #include <sycl/ext/intel/fpga_extensions.hpp>
```

Comment:
I think this change can be omitted completely, because it is quite useless if not upstreamed

Reply:
Removed